packages/commonwealth/client/scripts/views/modals/SessionRevalidationModal/SessionRevalidationModal.tsx

```diff
@@ -52,27 +53,36 @@ const SessionRevalidationModal = ({
 
       // if user tries to sign in with different address than
       // expected for session key revalidation
-      if (signedAddress !== walletAddress) {
+      if (
+        signedAddress === walletAddress ||
+        (app.user.activeAccounts.find((addr) => addr.address === walletAddress)
+          .community.ss58Prefix &&
+          addressSwapper({ address: walletAddress, currentPrefix: 42 }) ===
+            signedAddress)
```

Comment:
Would `ss58Prefix` be defined for all chain bases? I filtered the existing communities from the prod site and none of them seem to have `ss58Prefix` defined, in what scenarios would this logic get triggered?

Reply:
Did we make a change to Substrate chains? Edgeware, Clover, Darwinia, HydraDX, etc all have an ss58prefix set on my db snapshot.

Reply:
I don't have prod db access so I checked here -> `https://commonwealth.im/api/communities`. This returns all the available communities and atm none of them seem to have `ss58Prefix`

Reply:
Not sure about db changes that might have effected `ss58Prefix`.

Reply:
There should be some further down if you search for `"ss58_prefix":1`... it's just formatted as a number here so easy to miss...

Reply:
ah mb, i just found some. Sry for the back and forth.

Reply:
@raykyri i still think this logic will cause issues. If a single address is connected to multiple communities, the `activeAccounts` has multiple entries for each one of those. If one of those addresses is connected to 2 substrate based communities, the `ss58_prefix` would be different for both ex: `ss58_prefix` for `hydradx` is `63` and `plasm network` is `5`

Reply:
We're just checking if an ss58 prefix exists here, yeah - whatever it is we use ss58Prefix = 42 to reencode it.